code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -38,8 +42,12 @@ Borg Hypospray
 /obj/item/reagent_containers/borghypo/Initialize(mapload)
 	. = ..()
 
-	for(var/R in reagent_ids)
-		add_reagent(R)
+	for(var/reagent_id in reagent_ids)
+		add_reagent(reagent_id)
+
+	if(additional_reagent_ids)
```

Comment:
No need for this if

Reply:
Would we not want to check if the list is initialized before attempting to iterate over it?